proto/pdpb.proto

```diff
@@ -121,6 +121,7 @@ message TsoRequest {
 message Timestamp {
     int64 physical = 1;
     int64 logical = 2;
+    uint32 suffix_bits = 3;
```

Comment:
Could you please add some annotation for this param?

Reply:
Nice idea.